docs/cri/development.md

```diff
@@ -0,0 +1,40 @@
+# Getting Started for Developers
+
+This document provides steps to build `cri` plugin and test it locally. 
+
+Note that this is part of the migration from: https://github.com/containerd/cri#getting-started-for-developers
+
+## Install Dependencies
+
+1. Follow this [guide](https://github.com/containerd/containerd/blob/master/BUILDING.md) and install required dependencies.
+2. Install [CNI](https://github.com/containernetworking/cni) by following its homepage.
+
+## Build and Install cri
+
+When building the containerd, cri is automatically being built unless you specify no_cri build tag. For more detail, see this [guide](https://github.com/containerd/containerd/blob/master/BUILDING.md#build-containerd)
+
+### Running a Kubernetes local cluster
+
+If you already have a working development environment for supported Kubernetes
+version, you can try `cri` in a local cluster:
+
+0. If containerd is already running as a daemon service, you need to stop it first.
+```bash
+sudo service containerd stop
+```
+
+1. Start the version of `containerd` with `cri` plugin that you built and installed
+above as root in a first terminal:
+```bash
+sudo containerd -l debug
+```
+2. From the Kubernetes project directory startup a local cluster using `containerd`:
```

Comment:
We can point here to `crictl` as well

Reply:
If you mean verify the containerd is up with crictl, I agree and will put it here.

Reply:
verify/pointing to crictl repo anything will be ok.